src/app/dkim/dkim.component.html

```diff
@@ -88,7 +88,10 @@ <h2 class='subheading-2'>{{domain}}</h2>
                             </div>
                         </mat-grid-tile>
                         <mat-grid-tile style='border-bottom: 1px dotted #dedede; border-left: 1px dotted #dedede'><div class="grid_align_left">{{k.is_active ? 'Yes' : 'No'}}</div></mat-grid-tile>
-                        <mat-grid-tile style='border-bottom: 1px dotted #dedede; border-left: 1px dotted #dedede'><div class="grid_align_left">{{k.is_active && keys[0].date_active ? keys[0].date_active : '--'}}</div></mat-grid-tile>
+                        <mat-grid-tile style='border-bottom: 1px dotted #dedede; border-left: 1px dotted #dedede'><div class="grid_align_left">{{
```

Comment:
This CSS style could probably be extracted into a class, to not be repeated like this each time :)